src/components/dropdown/dropdown.tsx

```diff
@@ -4,12 +4,46 @@ import { ChevronDown, X, ChevronRight } from 'lucide-react'
 import { cn } from '@/lib/utils'
 import { HeaderTitle, Typography } from '@/components/Typography'
 import { useClickOutside } from '@/shared/hooks/useClickOutside'
-import { dropdownItemsData } from './data'
+import { DropdownItem } from './data'
 
 /**
- * "Prepare Your Proposal" custom dropdown menu with smooth animation
+ * Custom dropdown menu with smooth animation
  */
-export function PrepareProposalDropdown({ className, ...props }: HTMLAttributes<HTMLDivElement>) {
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  title: string
+  description: string
+  itemsData: DropdownItem[]
+}
+
+interface IDropdownItemComponent extends Omit<DropdownItem, 'linkUrl'> {
+  onClick: () => void
+}
+
+export const DropdownItemComponent = ({ id, onClick, title, text, Icon }: IDropdownItemComponent) => {
```

Comment:
`DropdownItem` is shorter and simpler. But the rest is good. I'm approving this.